python_modules/dagster/dagster/_core/storage/runs/sql_run_storage.py

```diff
@@ -948,6 +948,28 @@ def get_backfills(
             ]
         return backfill_candidates
 
+    def get_backfills_count(self, filters: Optional[BulkActionsFilter] = None) -> int:
+        subquery = db_subquery(self._backfills_query(filters=filters))
+        if filters.tags:
+            # runs can have more tags than the backfill that launched them. Since we filtered tags by
+            # querying for runs with those tags, we need to do an additional check that the backfills
+            # also have the requested tags. This requires fetching the backfills from the db and filtering them
+            rows = self.fetchall(subquery)
```

Comment:
I'd prefer to not have to fetch and deserialize all of the matching backfills just to check that they actually match the tags we are filtering on. But I'm not sure how else to do this